md/probes/jlink.md

```diff
@@ -0,0 +1,27 @@
+# JLink
+
+The JLink is a debug probe from [Segger](https://www.segger.com/). It is available as a stand-alone
+product, but also integrated into evaluation boards directly.
+
+Due to the proprietary nature of the JLink, probe-rs will not achieve the same speed as the official JLink tools.
```

Comment:
Well this is not really true, eh? Or is it? I mean it is mostly our shitty implementation. The protocol is really just barebones SWD.

Or am I in the wrong here?

Reply:
Of course we can improve our implementation, but the J-Link also has a higher level API which is not publicly documented. Similar to the high level API of the ST-Link. Because of this we will probably never achieve the same speed as the Segger tools.

Reply:
Ah okay, I didn't know that, thank you!